src/app/loadout/Loadout.tsx

```diff
@@ -0,0 +1,127 @@
+import ClassIcon from 'app/dim-ui/ClassIcon';
+import { t } from 'app/i18next-t';
+import { allItemsSelector, bucketsSelector } from 'app/inventory/selectors';
+import { DimStore } from 'app/inventory/store-types';
+import { getItemsFromLoadoutItems } from 'app/loadout-drawer/loadout-item-conversion';
+import { Loadout } from 'app/loadout-drawer/loadout-types';
+import { getLight, getModsFromLoadout } from 'app/loadout-drawer/loadout-utils';
+import { useD2Definitions } from 'app/manifest/selectors';
+import { AppIcon, faExclamationTriangle } from 'app/shell/icons';
+import { itemCanBeEquippedBy } from 'app/utils/item-utils';
+import clsx from 'clsx';
+import { BucketHashes } from 'data/d2/generated-enums';
+import _ from 'lodash';
+import React, { ReactNode, useMemo, useState } from 'react';
+import ReactDOM from 'react-dom';
+import { useSelector } from 'react-redux';
+import ItemCategorySection from './loadout-ui/ItemCategorySection';
+import PlugDef from './loadout-ui/PlugDef';
+import Subclass from './loadout-ui/Subclass';
+import styles from './Loadout.m.scss';
+import ModAssignmentDrawer from './mod-assignment-drawer/ModAssignmentDrawer';
+import { createGetModRenderKey } from './mod-utils';
+
+export default function Loadout({
```

Comment:
Would be nice to just rename this (and the file) to something that doesn't clash with Loadout

Reply:
Yeah will do, I couldnt think of a nice name at the time and its really just displaying a Loadout in its entirety. As I still can't think of something I think I am just going to go with `LoadoutDisplay` unless you have a better idea.